### PR TITLE
220 aug22

### PR DESCRIPTION
**Pull request by arjunguha**

### CODE REVIEW

File: stopify/package.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
     "source-map": "^0.6.0",
     "source-map-support": "^0.5.6",
     "sprintf": "^0.1.5",
-    "stopify-continuations": "0.5.0",
+    "stopify-continuations": "0.5.0-elementaryJS",
```
Comment (arjunguha) [inline stopify/package.json]: Yikes, good catch.

**Comment by arjunguha** [comment]
@baxtersa Now that master is quite far ahead of this branch, I think we should merge some commits in. Specifically, these are fixes that are probably relevant to master:

57e1de2
716afd6
f6a3e26
77713f0

Would you bring them in? I'd like to try to rebase the 220 changes here (and they are hacks that we need) onto master.

**Comment by baxtersa** [comment]
@arjunguha , I rebased this branch onto latest `master`. I imagine we don't want to land the HOF changes (they look ElementaryJS-specific) or the version updates, but everything else looks good.

How do you want to go about merging the desired commits into `master`? I can cherrypick them into a new PR, or we can just revert them on this PR.

**Comment by arjunguha** [comment]
Please do not rebase this branch (or at least keep around an old copy). I don't want the changes to master in this semester's 220. Cherry-picking the relevant commits will do:

57e1de2
716afd6
f6a3e26
77713f0

If you can't undo, let me know and I'll get someone to push their local copy.

**Comment by baxtersa** [comment]
Ah, sorry. I misunderstood the intention. I have reverted the rebase (someone may want to double check by diffing their local copy, but I reset it to what I believe is the right SHA from my `git reflog`). I'll create a new PR with cherrypicked commits.

**Comment by arjunguha** [comment]
lgtm

**Comment by baxtersa** [comment]
These commits have been cherrypicked and merged into `master`. Closing this PR now. Feel free to open a new one whenever more things to be merged are ready.

**Pull request closed without merging**